src/components/Logo/AnimatedLogo.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import s from './styles.module.scss';
+
+export const AnimatedLogo = props => {
+  const { classnames } = props;
+  return (
+    <div className={classnames(s.modusLogoSpin, s.logoSpin, s.logoBlack)}>
+      <svg
+        width="68"
+        height="50"
+        xmlns="http://www.w3.org/2000/svg"
+        className="svg-logo"
+        viewBox="-105 -10 420 140"
+      >
+        <g transform="translate(-1 -1)" fill="none" fillRule="evenodd">
+          <path
+            className={classnames(s.rightfill)}
```

Comment:
Using `classnames` only makes sense if you programmatically construct CSS classes. This files doesn't do that. You should be able to use static class names and work with CSS.

Reply:
Should i use template literals for all the variables ?

Reply:
No sir, CSS should be enough 😄 Even SCSS is ok 😄 

```jsx
<path className={s.myPath} />
```

```scss
.allPaths {
 fill: red;
}

.myPath {
  @extend .allPaths;
  stroke: blue;
}
```

Reply:
So using `classnames` to concatenate two classes is ok ?